src/mmw/sass/utils/_variables.scss

```diff
@@ -43,28 +43,84 @@ $sidebar-tab-subheader-height: 60px;
 //Border-radius
 $radius-sm: 2px;
 
-//NLCD
-$nlcd-11: #5475A8;
-$nlcd-21: #E8D1D1;
-$nlcd-22: #E29E8C;
-$nlcd-23: #ff0000;
-$nlcd-24: #B50000;
-$nlcd-31: #D2CDC0;
-$nlcd-41: #85C77E;
-$nlcd-52: #DCCA8F;
-$nlcd-71: #FDE9AA;
-$nlcd-81: #FBF65D;
-$nlcd-82: #CA9146;
-$nlcd-90: #C8E6F8;
+$nlcdColors: (
+    11: #5475A8,
+    12: #5475A8,
```

Comment:
The actual color values for the previously missing entries (skipping Alaska-only types) https://www.mrlc.gov/nlcd11_leg.php
```
12: #ffffff
42: #38814E
43: #D4E7B0
95: #64B3D5
```

These will match the NLCD overlay